_layouts/posts.html

```diff
@@ -15,6 +15,13 @@
 <body>
 <main>
   <p class="no-print"><a href="{{ site.baseurl }}/">< Back to Home</a></p>
+  {% if page.tags %}
+  <div class="tags">
+    {% for tag in page.tags %}
+    <a href='{{ site.baseurl }}/tag/{{tag}}'>#{{tag}}</a>
```

Comment:
I'd just wait for #34 to ship before merging this one. This change is the major reason for this PR.